### ISSUE TITLE
how to predict answer from question-evidence pair?

### ISSUE THREAD

**Issue by colinsongf** [YeliangLi/Chinese-QASystem#1]
how to predict answer from question-evidence pair?
how to output answers?

**Comment by YeliangLi** [comment]
In the modules.py file, the  class CRF has a viterbi_decode function. Use it to return a predicted tag sequence and the answers are identified by the "B", "I". @colinsongf

**Comment by colinsongf** [comment]
my mean is , if i have question-evidence pair, how to restore model ,and predict answers? In fact , a complete project have three parts: train\test, and predict answers line by line or a demo .
besides,a detailed readme file is also need to share more information.

**Comment by YeliangLi** [comment]
In file "train.py",it implemented model training and answer generation .In fact, I used the tf.data.TFRecordDataset to load training data and test data respectively. So you can see that I created two datasets in 2 loops  and used multiple lists to record question,evidence,targets_tags,q_e_comm_feas,e_e_comm_feas,question_length,evidence_length respectively.The  code placed in "with tf.device("/gpu:0")" is to use the model to process the training data and test data respectively and generate the loss and viterbi decoding result.

@colinsongf
